yarn.lock

```diff
@@ -1569,7 +1569,7 @@ eslint-plugin-react@7.7.0:
     prop-types "^15.6.0"
 
 "eslint-plugin-shopify@file:./.":
-  version "22.1.0"
+  version "23.0.0"
```

Comment:
`yarn install` on a fresh clone of master updates this. Our lock file is our of sync.